crowdsec-docs/unversioned/bouncers/node-js.mdx

```diff
@@ -0,0 +1,61 @@
+---
+id: node-js
+title: Node.js
+sidebar_position: 1
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/crowdsecurity/nodejs-cs-bouncer/main/docs/assets/app-sec.webp" width="400px" align="center" alt="Crowdsec Bouncer logo" />
+</p>
+
+## Overview
+This bouncer allows you to protect your Node.js application from IPs that have been detected by CrowdSec. Depending on the decision taken by CrowdSec, user will either get denied (403) or have to fill a captcha (401).
+
+It supports ban and captcha remediation, and all decisions with Ip or Range scope.
+
+## Requirements
+
+-   `Node.js >= 20`
+
+## Installation
+
+```bash
+npm install @crowdsec/nodejs-bouncer
+```
+
+## Features 
+
+- CrowdSec Local API support
+
+    - Handle `ip` and `range` scoped decisions
+    - `Live mode` or `Stream mode`
+
+- Support IpV4 and Ipv6 (Ipv6 range decisions are yet only supported in `Live mode`)
+
+## Usage
```

Comment:
Either we make this doc page a "clone" of https://github.com/crowdsecurity/nodejs-cs-bouncer/blob/main/docs/USER_GUIDE.md

or we mention the userguide with a link here